components/picker/stories/picker.stories.js

```diff
@@ -35,6 +37,15 @@ export default {
 			options: ["top", "left"],
 			control: { type: "select" },
 		},
+		withSwitch: {
+			name: "Display with a switch component",
```

Comment:
Do we want this to show in the control table, or is this just intended for Chromatic? It looks a bit odd when the picker label is on the top. We could also maybe make this control only show in the table when picker label is "left." Just throwing ideas out there :)